model/src/main/java/org/mskcc/cmo/metadb/model/Run.java

```diff
@@ -5,6 +5,7 @@
 import java.util.Collections;
 import java.util.List;
 import org.apache.commons.lang.builder.ToStringBuilder;
+import org.mskcc.cmo.metadb.model.igo.IgoRun;
```

Comment:
Considering removing `Run` completely pending decision on whether we want to change the type of `sampleMetadata.libraries.runs` to a `String` since that is how we store this member in the graph database anyway.